src/udev/udevadm-trigger.c

```diff
@@ -86,35 +100,77 @@ static int exec_list(sd_device_enumerator *e, sd_device_action_t action, Set **s
                         continue;
                 }
 
-                if (settle_set) {
-                        r = set_put_strdup(settle_set, syspath);
+                /* If the user asked for it, write event UUID to stdout */
+                if (arg_uuid)
+                        printf(SD_ID128_FORMAT_STR "\n", SD_ID128_FORMAT_VAL(id));
```

Comment:
It is better to include syspath? It may be useful when multiple devices are triggered.

Reply:
we aready output that with --verbose, see further up. So my thinking was that if you want both, specify "--verbose --uuid", but if you want just one of the two use the switch you need.

Reply:
Oh, indeed. Thanks.